packages/fluentui/react-bindings/src/compose/appendClasses.ts

```diff
@@ -0,0 +1,10 @@
+export function appendClasses(...classes: string[]) {
+  let result = '';
+  for (const className of classes) {
+    if (className) {
+      result = (result ? `${result} ` : '') + className;
+    }
+  }
+
+  return result;
```

Comment:
I know this PR is just moving things for what I understood. But wouldn't this code be equivalent to:

`return classes.join(' ')` ?